pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/BKCluster.java

```diff
@@ -201,13 +201,13 @@ protected void cleanupTempDirs() throws Exception {
     private ServerConfiguration newServerConfiguration(int index) throws Exception {
         File dataDir;
         if (clusterConf.dataDir != null) {
-            dataDir = new File(clusterConf.dataDir);
+            dataDir = new File(clusterConf.dataDir + "/" + index);
```

Comment:
Will this break the existing pulsar standalone?

Reply:
Indeed, I updated the code to be compatible with the original data directory.